cmd/shipper/main.go

```diff
@@ -166,14 +195,14 @@ func main() {
 				EnvVars: []string{"SHIPPER_COMMIT_MESSAGE"},
 				Value:   "Deploy",
 			},
-			&cli.StringFlag{
+			&cli.StringSliceFlag{
 				Name:     "container-image",
 				Aliases:  []string{"ci"},
 				Usage:    "Container image",
 				EnvVars:  []string{"SHIPPER_CONTAINER_IMAGE"},
```

Comment:
```suggestion
				EnvVars:  []string{"SHIPPER_CONTAINER_IMAGES"},
```

I assume this becomes a comma-separated list? e.g. `SHIPPER_CONTAINER_IMAGES=image-1,image-2`